templates/includes/sidebar.html

```diff
@@ -1,4 +1,17 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>
+      User: {{ user.username }}
+    </li>
+    <li>
+    <a href="{% url 'logout' %}">Logged out</a>
```

Comment:
The text for the logout link should be 'Logout' instead of 'Logged out' to correctly indicate the action to the user.